### ISSUE TITLE
Scoping issue with enum values

### ISSUE THREAD

**Issue by cristianbuse** [WaynePhillipsEA/twinbasic#150]
**Describe the bug**
Bitwise ```Or``` operator in an Enum is not evaluated in TB

**To Reproduce**
```VBA
Private Enum EXT_WND_STYLES
    WS_EX_CLIENTEDGE = &H200
    WS_EX_WINDOWEDGE = &H100
    WS_EX_OVERLAPPEDWINDOW = WS_EX_WINDOWEDGE Or WS_EX_CLIENTEDGE 'Cannot be evaluated in TB
End Enum
```

**Expected behavior**
Code is valid and works in VBA. Same code should compile and work in TB.

**Screenshots**
![test](https://i.stack.imgur.com/gE3cT.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise V1909
 - twinBASIC compiler v0.9.2720

**Additional context**
None

**Comment by WaynePhillipsEA** [comment]
Hmm... did you have those enum values defined in another Enum by chance?   

Your example works fine here, but I notice that if you have the same named enum values in two distinct enums, this error occurs.   We will get that fixed, as tB should be prioritizing the bindings to the current enum scope before looking at the global scope.

**Comment by cristianbuse** [comment]
No. Same Enum. Same error occurs when using a + on a previous value:
```VBA
Private Enum TEST
    val1 = 1
    val2 = val1 + 1 'Error here
End Enum
````

**Comment by WaynePhillipsEA** [comment]
Sorry, yes, confirmed on a private enum.   Works on a public enum, due to the global scoping.  It's the same issue I mentioned above since the **current** enum values are not being considered in the resolving unless they happen to be in the global scope (public).   We'll get it fixed, thanks for reporting.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.509, thanks for reporting
